manifests/kustomize/env/aws/config

```diff
@@ -0,0 +1,23 @@
+{
+namespace: $(kfp-namespace),
+executorImage: gcr.io/ml-pipeline/argoexec:v2.7.5-license-compliance,
+containerRuntimeExecutor: $(kfp-container-runtime-executor),
+artifactRepository:
+{
+    s3: {
+        bucket: $(kfp-artifact-bucket-name),
+        keyPrefix: artifacts,
+        endpoint: s3.amazonaws.com,
+        insecure: true,
```

Comment:
`insecure` should be false as I remembered? Can you check if your S3 has persistent logs?

Reply:
https://github.com/kubeflow/pipelines/blob/master/manifests/kustomize/base/argo/workflow-controller-configmap.yaml#L17 

```
 insecure: true
```

And I checked my S3 and it has persistent logs.

Reply:
Ok. Looks insecure works as well